coolwsd.xml.in

```diff
@@ -204,6 +204,18 @@
             <locking desc="Locking settings">
                 <refresh desc="How frequently we should re-acquire a lock with the storage server, in seconds (default 15 mins) or 0 for no refresh" type="int" default="900">900</refresh>
             </locking>
+
+            <aliases desc="Defines which hostnames are actually pointing to the same instance of the WOPI host.">
+                <fallback desc="fallback setting when alias is missing from the list.
+                                allowonce attribute will be taken into consideration only when there are no hosts and alias pair defined in config,
+                                it will allow first host that gets connected and denied rest of the hosts. allow attribute is used to allow/deny
+                                host/alias missing from the config" allow="false" allowonce="true" />
```

Comment:
Whenever we have two-word identifiers, we use underscore, like `user_interface`. Let's have `allow_once`, which is easier to read and more consistent.